packages/kununu-footer/index.jsx

```diff
@@ -84,7 +90,6 @@ export default class Footer extends Component { // eslint-disable-line
               </div>
               ),
             )}
-
             {tuv ?
```

Comment:
you could write it like tuv && <div ...>, then you don't need the ternary fallback

Reply:
This will return false in the dom. You can only do this if you do something like tuv === 1 or something.

Reply:
🤔

Reply:
Kaitlin is right here, if `tuv` is false it will return false on the dom.
it should be like this `{tuv && <div /> || null}` where is the same like `{tuv ? <div /> : null}`.
I have seen "false" text in the tests so please be carefull with this.
Also I have seen empty lines in the tests (where they are looking like bugs for me), these lines are generated from the returned empty strings, as it happened here. Null should be returned in this case, it is accepted by React (not undefined). The same happens in the dom, not only this the tests. So :child selector might be confused with these empty strings.